src/AuthenticatedAPIClient/authInterface.js

```diff
@@ -101,8 +105,35 @@ export default function applyAuthInterface(httpClient, authConfig) {
       // Attempt to refresh the JWT cookies.
       return httpClient.refreshAccessToken()
         // Successfully refreshed the JWT cookies, fire the callback function.
-        .then(() => {
-          callback(httpClient.getDecodedAccessToken());
+        .then((response) => {
+          const refreshedAccessToken = httpClient.getDecodedAccessToken();
+
+          /* istanbul ignore next */
+          if (refreshedAccessToken === null) {
+            // This is the success block for refreshing an access token.
+            // Sometimes the access token is null for an unknown reason.
+            // Log here to learn more.
+            if (httpClient.loggingService && httpClient.loggingService.logError) {
+              httpClient.loggingService.logError(
+                new Error('Access token is null after refresh.'),
+                {
+                  previousAccessToken: accessToken,
+                  axiosResponse: response,
+                },
+              );
+
+              // Wait 50ms and check again. Maybe we're in a race to set the cookie?
+              setTimeout(() => {
+                const checkAccessToken = httpClient.getDecodedAccessToken();
```

Comment:
1. Since we know the callback token is null, this could be added here, or directly in the app (like the profile app where I have that other logError call.  The advantage of adding it here is if you want to push this to multiple apps to get more calls that might fail.  The advantage of adding to the app is that we can experiment and push out changes faster.
2. Nit: maybe name this `delayedRefreshAccessToken`?
3. Could you change this to a logInfo call with a message we could search for, like 'Access token check after 50ms after null refresh token: `${delayedRefreshAccessToken !== null}`.  You could send this same `delayedRefreshAccessToken !== null` result in the custom attributes as well.  Please call this logInfo message whether the token is null or not, so we can see a success or a failure (or possibly nothing, but hopefully not).
4. I could imagine a short timeout like this being good, and I can imagine a longer timeout (like 500ms) being something work trying.  Maybe just add 2 setTimeout calls (whether here or in the profile app)?  Note: this is an error condition anyway, so not a big deal if we slow it down a little.